server/routes/api/posts.js

```diff
@@ -0,0 +1,102 @@
+const Posts = require('../../models/forums/Post');
+
```

Comment:
Just as you have included the model Posts, you need to include the Users model.

`const Users = require(../../models/User);`  or the equivalent path.